packages/components/src/sites-table/use-sites-table-sorting.tsx

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line no-restricted-imports
+import type { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+
+interface SitesTableSortOptions {
+	sort?: string;
```

Comment:
Maybe `sortKey`?

```suggestion
	sortKey?: 'updated-at';
```

Reply:
That makes sense. I renamed `order` to `sortOrder` to keep the naming convention consistent.